xonsh/built_ins.py

```diff
@@ -555,6 +564,16 @@ def run_subproc(cmds, captured=True):
                 except PermissionError:
                     e = 'xonsh: subprocess mode: permission denied: {0}'
                     raise XonshError(e.format(cmd[0]))
+        _stdin_file = None
+        if captured == 'object' and stdin is not None:
+            _stdin_file = tempfile.NamedTemporaryFile()
+            cproc = Popen(['cat'],
+                          stdin=stdin,
+                          stdout=PIPE)
+            tproc = Popen(['tee', _stdin_file.name],
```

Comment:
The UNIX tools from "git for windows" are not the path by default. I guess most people don't select the option to add them to the path.